Source/Core/Common/VR/VRInput.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "VRBase.h"
```

Comment:
```suggestion
#include "Common/VR/VRBase.h"
```

Same for other files.